getting-started/templates/systemlink-values.yaml

```diff
@@ -517,6 +539,26 @@ dashboardhost:
   ## for more documentation and examples for these values.
   ##
   grafana:
+    ## Configuring extra containers for the Dashboardhost Service.
+    ##
+    ## @param extraContainers Additional containers to run in the same pod as the Dashboardhost Service.
+    ## It can be used for authentication proxy containers, such as the Cloud SQL Auth Proxy.
+    ## ref: https://cloud.google.com/sql/docs/postgres/connect-kubernetes-engine#run_the_in_a_sidecar_pattern
+    extraContainers: ""
```

Comment:
Please mention why a different name is used for grafana in the PR description

Reply:
Added